### PR TITLE
Added --agent-id to dcos task list.

### PR DESCRIPTION
**Pull request by armandgrillet**

### CODE REVIEW

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

File: python/lib/dcoscli/tests/integrations/test_task.py [hunk 1]
```diff
@@ -95,18 +95,31 @@ def test_task_completed():
...
+    returncode, stdout, _ = exec_command(
+        ['dcos', 'task', 'list', 'test-app2', '--json'])
+
+    assert returncode == 0
+
+    agent = json.loads(stdout.decode('utf-8'))[0]['slave_id']
+    assert_lines(
+        ['dcos', 'task', 'list', 'test-app2', '--json', '--agent-id', agent],
+        1,
+        greater_than=True)
```
Comment (bamarni) [inline python/lib/dcoscli/tests/integrations/test_task.py]: Can we iterate over the JSON and assert that all tasks have the expected `slave_id`?

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Review by bamarni — APPROVED** [review]

**Comment by armandgrillet** [comment]
@mesosphere-ci run linux integration tests

**Pull request merged**